docs/_sass/_main.scss

```diff
@@ -410,11 +413,11 @@ button {
     padding: 28px;
     font-weight: bold;
```

Comment:
Let's change any instance of `bold` to be 700 so it maps back to the font family declarations correctly.